src/formatters/needlessDisablesStringFormatter.js

```diff
@@ -0,0 +1,25 @@
+import chalk from "chalk"
+import path from "path"
+
+function logFrom(fromValue) {
+  if (fromValue.charAt(0) === "<") return fromValue
+  return path.relative(process.cwd(), fromValue).split(path.sep).join("/")
+}
+
+export default function (report) {
+  let output = ""
+
+  report.forEach(sourceReport => {
```

Comment:
Yeah, we should do that. I think I did it on purpose but was tired and not thinking straight :)